server/models/services/CollectionJobService.js

```diff
@@ -106,26 +101,8 @@ const createCollectionJob = async (
     options
 ) => {
     if (!testPlanRun) {
-        let user = await getUserByUsername(responseCollectionUser.username);
-        if (!user) {
-            const roles = [{ name: User.TESTER }];
-            user = await createUser(
```

Comment:
It would have been easy to persist this obviated logic, though I can only imagine the kind of confusion it might have caused in some future exceptional circumstance. Thanks for being thorough and removing it!